production.py

```diff
@@ -17,7 +17,10 @@ class ProductionTemplate(ModelSQL, ModelView):
     __name__ = 'production.template'
 
     name = fields.Char('Name', required=True)
-    uom = fields.Many2One('product.uom', 'Uom', required=True)
```

Comment:
Pq? si es "required" com atribut, tenim a nivell SQL "not null". Si ho fem com estat, no tenim control via SQL.

Si es manté com states, caldria eliminar el "not null" a la base de dades. Es pot fer via SQL directament a la base de dades. No cal el __register__